page_creation.rst

```diff
@@ -262,6 +262,20 @@ to get your *new* lucky number!
 
     http://localhost:8000/lucky/number
 
+Now you may wonder where the Web Debug Toolbar has gone: that's because we don't write
+a ``body`` element using the current template. If we incorporate the default
```

Comment:
`a body element` -> `a <body> HTML tag` ?

Reply:
Yes, though, strictly speaking, [a tag is not an element](http://perfectionkills.com/tag-is-not-an-element-or-is-it/), and I guess the element should be present, not only an opening tag.

Reply:
Actually, only `</body>` have to be present. Not even a complete body element.

Strictly speaking there is a body element (The node in a DOM tree), which consists of an open and close tag. 

In any case, let's be explict here and say "that's because there is no `</body>` tag in the current template. You can add the body element yourself, or extend `base.html.twig`, which contains all default HTML elements." (also removing first person usage)

If you agree, can you please update your PR (otherwise, just comment back)? Thanks!